### PR TITLE
Modify: Google.Chrome version 126.0.6478.127

### PR DESCRIPTION
**Pull request by aisgbnok**
Checklist for Pull Requests
- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] This PR only modifies one (1) manifest
- [X] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [X] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

- Adds `arm64` architecture

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/160397)

### CODE REVIEW

**Comment by aisgbnok** [comment]
@SpecterShell, `Google.Chrome` seems to be handled by your automation. Just confirming that `arm64` was not left out for a reason? Is this something you would consider adding for further updates? Thank you.

https://github.com/SpecterShell/Dumplings/blob/main/Tasks/Google.Chrome/Script.ps1

Same for `Google.Chrome.Beta`:
- #160398
- https://github.com/SpecterShell/Dumplings/blob/main/Tasks/Google.Chrome.Beta/Script.ps1

**Comment by SpecterShell** [comment]
> @SpecterShell, `Google.Chrome` seems to be handled by your automation. Just confirming that `arm64` was not left out for a reason? Is this something you would consider adding for future updates? Thank you.
> 
> [SpecterShell/Dumplings@`main`/Tasks/Google.Chrome/Script.ps1](https://github.com/SpecterShell/Dumplings/blob/main/Tasks/Google.Chrome/Script.ps1?rgh-link-date=2024-06-27T18%3A14%3A07Z)
> 
> Same for `Google.Chrome.Beta`:
> 
>     * [Modify: Google.Chrome.Beta version 127.0.6533.26 #160398](https://github.com/microsoft/winget-pkgs/pull/160398)
> 
>     * [SpecterShell/Dumplings@`main`/Tasks/Google.Chrome.Beta/Script.ps1](https://github.com/SpecterShell/Dumplings/blob/main/Tasks/Google.Chrome.Beta/Script.ps1?rgh-link-date=2024-06-27T18%3A14%3A07Z)
> 
> 
> `Google.Chrome.Dev`, may not have `arm64` builds yet.

`arm64` was not available for the enterprise msi when I was working on the automation of this package. It was only available for the EXE counterparts. I will update my automation soon.

File: manifests/g/Google/Chrome/126.0.6478.127/Google.Chrome.installer.yaml [hunk 1]
```diff
@@ -31,6 +31,12 @@ Installers:
   AppsAndFeaturesEntries:
   - ProductCode: '{A6A8A820-1396-378F-ABE5-4EBDEA19FE69}'
     UpgradeCode: '{C1DFDF69-5945-32F2-A35E-EE94C99C7CF4}'
+- Architecture: arm64
+  InstallerUrl: https://dl.google.com/dl/chrome/install/googlechromestandaloneenterprise_arm64.msi
+  InstallerSha256: 131227BD386C8EBB9928C2244E6960A8BCC15449D64D7AE53E0DFAA65E0CAAE9
+  AppsAndFeaturesEntries:
+    - ProductCode: '{13ED2040-F468-3604-8F62-AB0777C55B41}'
+      UpgradeCode: '{C1DFDF69-5945-32F2-A35E-EE94C99C7CF4}'
```
Comment (SpecterShell) [inline manifests/g/Google/Chrome/126.0.6478.127/Google.Chrome.installer.yaml:39]:
```suggestion
  ProductCode: '{13ED2040-F468-3604-8F62-AB0777C55B41}'
  AppsAndFeaturesEntries:
  - ProductCode: '{13ED2040-F468-3604-8F62-AB0777C55B41}'
    UpgradeCode: '{C1DFDF69-5945-32F2-A35E-EE94C99C7CF4}'

```

File: manifests/g/Google/Chrome/126.0.6478.127/Google.Chrome.installer.yaml [hunk 2]
```diff
@@ -31,6 +31,14 @@ Installers:
...
   - ProductCode: '{A6A8A820-1396-378F-ABE5-4EBDEA19FE69}'
     UpgradeCode: '{C1DFDF69-5945-32F2-A35E-EE94C99C7CF4}'
+- Architecture: arm64
+  InstallerUrl: https://dl.google.com/dl/chrome/install/googlechromestandaloneenterprise_arm64.msi
+  InstallerSha256: 131227BD386C8EBB9928C2244E6960A8BCC15449D64D7AE53E0DFAA65E0CAAE9
+  ProductCode: '{13ED2040-F468-3604-8F62-AB0777C55B41}'
+  AppsAndFeaturesEntries:
+  - ProductCode: '{13ED2040-F468-3604-8F62-AB0777C55B41}'
+    UpgradeCode: '{C1DFDF69-5945-32F2-A35E-EE94C99C7CF4}'
+
```
Comment (aisgbnok) [inline manifests/g/Google/Chrome/126.0.6478.127/Google.Chrome.installer.yaml:41]:
```suggestion
```

**Review by aisgbnok — COMMENTED** [review]
Remove empty line

**Review by stephengillie — APPROVED** [review]

**Pull request merged**